joomla/joomla-cms
# [3.9.1] [com_fields] Make sure disabled fields are not added to the request at all

Pull Request for Issue #22038 & #22519

### Summary of Changes

Make sure disabled fields are not added to the request at all

### Testing Instructions

1) Create a custom field for your articles. 
2) Check that ACL is set to edit this field only at super user (this is the default) and set it for Editor to denied
3) create an Article as an Editor (Usergroup) 
4) You can't save the Article because of an Error "Invalid Field"

### Expected result

Error is gone

### Actual result

![image](https://user-images.githubusercontent.com/2596554/47937681-c93c9a00-dee1-11e8-88d9-2e94d9107ddc.png)


### Documentation Changes Required

none.

cc @laoneo Please let us know you opinion / technical insight here. As disabled should not be used for security we might also need to change more places too?
